qemu/tests/cfg/win_virtio_driver_install_by_installer.cfg

```diff
@@ -40,6 +40,21 @@
     nic_model_nic1 = rtl8139
     q35:
         nic_model_nic1 = e1000e
+    # install winfsp tool
+    i386, i686:
+        install_path = 'C:\Program Files'
+    x86_64:
+        install_path = 'C:\Program Files (x86)'
+    install_cmd = 'msiexec /i WIN_UTILS:\winfsp-1.9.21096.msi /qn'
+    check_installed_cmd = 'dir "%s" |findstr /I winfsp'
```

Comment:
will this pkg name(winfsp-1.9.21096.msi) change in future?
does it depend on the windows version? I mean, e.g.
win10:
name1
win2022:
name2

Reply:
for these two cmds, it may not be possible to change the cmds in different cases, so we can handle it in the same way, like params.get('install_cmd', 'msiexec xxxx')

Reply:
Then we can remove all the cmd params from cfg